src/utilities/theme/types.ts

```diff
@@ -22,14 +22,23 @@ export interface TopBar extends ThemeColor {
 }
 
 export type ThemeColors = {
-  topBar: TopBar;
+  topBar?: TopBar;
+  surface?: string;
+  onSurface?: string;
+  interactive?: string;
+  interactiveNeutral?: string;
+  branded?: string;
+  critical?: string;
+  warning?: string;
+  highlight?: string;
+  success?: string;
 };
 
 export interface Theme {
   /** Sets the logo for the top bar and contextual save bar components*/
   logo?: ThemeLogo;
   /** Sets the background color of the top bar component. Complimentary and typography colors are determined programmatically */
-  colors?: ThemeColors;
+  colors: ThemeColors;
```

Comment:
This would be considered a breaking change, so we'd want to ship this as part of a `v5.0.0` alpha.